### PR TITLE
Release v4.4.1

### PR DESCRIPTION
**Pull request by str4d**
Due to Homu merge confusion, the release commit for this release is
0dade79ce75480bc352d1c3e219409f5d2bc0c7e (the final commit in the
release branch), not the merge commit.

### CODE REVIEW

**Review by nuttycom — APPROVED** [review]
ACK

File: doc/release-notes/release-notes-4.4.1.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+Notable changes
+===============
+
+Fixed chain sync stall bug
+--------------------------
+
+The 4.3.0 release included a change to prevent redundant `getheaders` P2P
+requests, to reduce node bandwith usage. This behaviour could be disable by
```
Comment (daira) [inline doc/release-notes/release-notes-4.4.1.md:8]:
```suggestion
requests, to reduce node bandwith usage. This behaviour could be disabled by
```

**Review by daira — APPROVED** [review]
utACK

**Comment by str4d** [comment]
@zkbot r+

**Comment by str4d** [comment]
Transient CI failure.

@zkbot retry

**Pull request merged**